docs/rule/no-yield-block-params-to-else-inverse.md

```diff
@@ -0,0 +1,21 @@
+# no-yield-block-params-to-else-inverse
+
+Yielding to else block is mainly useful for supporting curly invocation syntax. However, the else block in curly invocation syntax does not support consuming block params.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+{{yield "some" "param" to="else"}}
+```
+
+```hbs
+{{yield "some" "param" to="inverse"}}
+```
+
+This rule **allows** the following:
+
+```hbs
+{{yield}}
+```
```

Comment:
`{{yield "some" "param"}}`, `{{yield to="else"}}` and `{{yield to="inverse"}}`

Reply:
Can you add these to "good" section?